cmd/pki/create.go

```diff
@@ -0,0 +1,100 @@
+/* -------------------------------------------------------------------------- *\
```

Comment:
I think following up on my suggestion (and a pattern defined in another issue/pr) we should probably have `cmd/pki/create.go` moved to `cmd/$NOUN/$VERB` if that makes sense?

If we want "pki" to be the "noun" then we should have the folder name match the command syntax.

Reply:
I merged #25 which calls out the structure I poorly articulated above. Should we use that structure for `ae pki create/delete`? Maybe `ae cert`? Unsure what you (or maybe @taniwha3?) would think is best.

Reply:
For the initial PR here, I was just following the description in #23. I also prefer "noun" + "verb".

In this specific case I prefer `ae cert create` over `ae pki create`.

What about `ae cert create --rootCA`, indicating that we want to create a rootCA?
For client certs we could then go with `ae cert create --user <name> --root ca.crt`.

I will have a look how `cfssl gencert` handles commands + flags. I guess it is a commonly used tool, so might be worth it to be consistent with it?